tests/unit/test_adapter.py

```diff
@@ -1,153 +1,41 @@
 import unittest
+import pytest
 from multiprocessing import get_context
 from unittest import mock
 
-import dbt.flags as flags
 from dbt.exceptions import DbtRuntimeError
 from agate import Row
 from pyhive import hive
 from dbt.adapters.spark import SparkAdapter, SparkRelation
 from .utils import config_from_parts_or_dicts
 
+pytest.mark.usefixtures("target_http")
```

Comment:
Do we need this if `target_http` is already marked `autouse=True`?
https://github.com/dbt-labs/dbt-spark/pull/1093/files#diff-e5f553b05b74e4001ce554a76b45ad0bdaee19b021d0a6cbbfbb9c884ed9d359R22

Reply:
No I just forgot to delete this one